fork/module-to-cdn/jsonschema.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
![image](https://github.com/Talend/ui/assets/19857479/9ac7b2a9-e258-46ba-a453-4082a2a0c7f1)
you can use this file to configure vscode and get completion over modules.json